### DISCUSSION TITLE
邮件内链接问题

### DISCUSSION THREAD

**Discussion by inkss** [ArtalkJS/Artalk#112] [Q&A] [answered]
![枋柚梓的猫会发光  您的文章 Volantis：自定义右键菜单实现 - 枋柚梓的猫会发光 有新回复_22-04-18_22-31-55](https://user-images.githubusercontent.com/31947043/163823363-11d8b78d-53d8-4e3d-8b5f-33744cd25c76.png)

实际应该为 `https://inkss.cn/blog/3717f151/#atk-comment-985`

**Comment by qwqcode** [comment, marked as answer]
[packages/artalk/src/components/list.ts ](https://github.com/ArtalkJS/Artalk/blob/master/packages/artalk/src/components/list.ts#L120)

这个是对的，不是 BUG

**Reply by qwqcode** [reply]
https://inkss.cn/blog/3717f151/?atk_comment 末尾应该带上斜杠吗

**Reply by inkss** [reply]
可是后面带 `/` 和不带的，不是不一样吗

**Reply by inkss** [reply]
嗯，得带上斜杠

**Reply by inkss** [reply]
数据库里存的也是带斜杠的地址

**Reply by qwqcode** [reply]
刚刚测了下，这个在后续的版本应该已经修复了，这个版本目前还未发布

**Reply by inkss** [reply]
厉害了，给你点赞

**Reply by inkss** [reply]
@qwqcode 我回复的为什么没有发邮件通知呀？

**Reply by qwqcode** [reply]
后端更新 v2.1.2

**Reply by inkss** [reply]
邮件内连接现在正常了，只是我回复的还是没有发邮件提醒。

notic 表里也只有一条记录，对方评论有邮件提示，回复这条评论却没有。

**Comment by inkss** [comment]
`Mozilla/5.0 (Windows NT 11.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36 Edg/100.0.1185.44`

是 edge 不是 chrome

**Reply by qwqcode** [reply]
Edge 是 chromium 套个壳的浏览器，所以 ua 这个判不太准，但可以加个修正

**Reply by qwqcode** [reply]
https://github.com/ArtalkJS/Artalk/commit/6c38a9048ddc93420ffe6a7d540f9c8af20893c9 已修复

**Reply by inkss** [reply]
辛苦，感觉这几天被我不堪其扰 emm

**Reply by inkss** [reply]
PS 后端 2.1.2 是发了两次吗？
